src/plugins/data_table/data_table.js

```diff
@@ -347,6 +347,11 @@ var LocalDataTable = (function() {
       }
     },
 
+    _columnElements: function(selector) {
+      var selectorString = selector || "";
+      return this.$("table").find("thead tr th" + selectorString);
+    },
```

Comment:
Yeah, I was using it in action table, but I felt that the method was better suited here instead. I'll make sure I had added tests for this